layers/+completion/compleseus/packages.el

```diff
@@ -36,24 +36,7 @@
     (selectrum :toggle (eq compleseus-engine 'selectrum))
     (vertico
      :toggle (eq compleseus-engine 'vertico)
-     ;; TODO remove when `vertico-repeat' on ELPA
```

Comment:
why did you remove the extension of vertico, they are not an elpa or are they?

Reply:
they are - the the extension files are part of the https://elpa.gnu.org/packages/vertico.html tar archive.

Reply:
thanks! it seems that they are not enabled by default, is it possible to add them again?